src/repror/cli/setup_rattler_build.py

```diff
@@ -14,7 +14,7 @@
 from rich.panel import Panel
 
 
-class SetupRattlerBuild(StrEnum):
+class SetupRattlerBuild(str, Enum):
```

Comment:
why not StrEnum?

Reply:
Because that trips up the type-checkers that it always thinks the value is a str and not an Enum. This should do basically the same :)